pkg/kubelet/pluginmanager/pluginwatcher/plugin_watcher_test.go

```diff
@@ -56,6 +57,14 @@ func init() {
 
 func cleanup(t *testing.T) {
 	require.NoError(t, os.RemoveAll(socketDir))
+	// Creating a different directory. os.RemoveAll is not atomic enough;
+	// os.MkdirAll can get into an "Access Denied" error on Windows.
+	d, err := ioutil.TempDir("", "plugin_test")
```

Comment:
We might want to refactor these tests rather than creatively using cleanup. (to explicitly get new state per-test, rather than relying on this cleanup)